packages/frontend/src/components/wallet-migration/Disable2FA.jsx

```diff
@@ -132,6 +133,12 @@ const Disable2FAModal = ({ handleSetActiveView, onClose }) => {
                     localDispatch({ type: ACTIONS.SET_CURRENT_DONE });
                 }
             } catch (e) {
+                dispatch(showCustomAlert({
+                    errorMessage: e.message,
+                    success: false,
+                    messageCodeHeader: 'error'
+                }));
+                await new Promise((r) => setTimeout(r, 3000));
```

Comment:
error shows for 3 seconds. Without this delay, error message goes away in a second.